.github/ISSUE_TEMPLATE/question.md

```diff
@@ -8,6 +8,14 @@ labels: ❔Question
 
 <!-- Run `react-native info` in your terminal and paste its contents here. -->
 
+## Way of upgrading
+
+I used:
+- [ ] [upgrade-helper](https://react-native-community.github.io/upgrade-helper/)
+- [ ] [cli](https://github.com/react-native-community/cli)
+- [ ] Other 
+<!-- If `Other`, please describe below what you used. -->
```

Comment:
This makes it feel a bit like a set of radio buttons. I.e. people can choose one or the other, whereas maybe you’d want to ensure people always check the diff before asking?

Reply:
I wanted to leave it open for people who just recreate a project instead of upgrading etc, but I get your point.

Reply:
If we put in the body something like "make sure you check the diff at..." usually people will just skip it, unfortunately

Reply:
How do we feel about that? This might be a bit nicer?